src/server/daily.ts

```diff
@@ -0,0 +1,117 @@
+import axios from "axios";
+import claimsAreValid from "../shared/jwt";
+import { DAILY_API_KEY, DAILY_STAGING } from "./env";
+
+const isStaging = DAILY_STAGING === "true";
+
+let dailyAPIDomain = "daily.co";
+if (isStaging) {
+  dailyAPIDomain = "staging.daily.co";
+}
+
+const dailyAPIURL = `https://api.${dailyAPIDomain}/v1`;
+
+// The data we'll expect to get from Daily on room creation.
+// Daily actually returns much more data, but these are the only
+// properties we'll be using.
+interface CreatedDailyRoomData {
+  id: string;
+  name: string;
+  url: string;
+}
+
+export async function createRoom(): Promise<CreatedDailyRoomData> {
```

Comment:
This is basically just a move operation from another file; no code has changed here, but ofc if you feel like giving it a pass any feedback is still welcome.